feature/container/containerz/tests/container_lifecycle/containerz_test.go

```diff
@@ -479,7 +480,12 @@ func TestVolumes(t *testing.T) {
 		// Allow time for removal to settle.
 		time.Sleep(5 * time.Second)
 
-		createdVolumeName, err := cli.CreateVolume(ctx, volumeName, "local", nil, nil)
+		mountOpts := map[string]string{
+			"options":    "bind",
```

Comment:
Are there other non-bind options that we should be testing?

Reply:
Currently no - this is the only option we use.